src/config.js

```diff
@@ -4,6 +4,65 @@ import testConfig from './config.test';
 
 const rpcHost = require('./env.json').rpcHost;
 
+const gnosis = {
+  tokenContract: '0x6810e776880C02933D47DB1b9fc05908e5386b96',
+  information: {
+    aliasName: 'Gnosis',
+    website: 'https://gnosis.pm/',
+    logo: 'https://gnosis.pm/favicon.ico?v=1',
+  },
+  events: {
+    BidSubmission: {
+      args: {
+        tokens: 'amount',
+        sender: 'sender',
+      },
+      customArgs: {
```

Comment:
There is no _from in `BidSubmission(receiver, amount);` actually tokens are claimed AFTER ICO ends. we support this by adding another event. check `claimTokens` and there is Transfer.... where you probably should filter by `_from`